content/en/containers/kubernetes/control_plane.md

```diff
@@ -326,40 +326,20 @@ Amazon Elastic Kubernetes Service (EKS) supports monitoring all control plane co
   - Datadog Operator >= `v1.13.0`
 - Enable the Datadog [Cluster Agent][6]
 
-{{< tabs >}}
-{{% tab "Datadog Operator" %}}
-
-Add the following annotations. Operator installations are limited to API Server [metrics][1]. For support for `kube_controller_manager` and `kube_scheduler` metrics, use the Helm install.
-
-```yaml
-annotations:
-  ad.datadoghq.com/endpoints.check_names: '["kube_apiserver_metrics"]'
-  ad.datadoghq.com/endpoints.init_configs: '[{}]'
-  ad.datadoghq.com/endpoints.instances:
-    '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
-```
-
-[1]: https://aws.github.io/aws-eks-best-practices/reliability/docs/controlplane/#monitor-control-plane-metrics
-{{% /tab %}}
-{{% tab "Helm" %}}
-
 Add the following annotations to the `default/kubernetes` service:
 
 ```yaml
 annotations:
   ad.datadoghq.com/endpoints.check_names: '["kube_apiserver_metrics"]'
   ad.datadoghq.com/endpoints.init_configs: '[{}]'
-  ad.datadoghq.com/endpoints.instances:
-    '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
+  ad.datadoghq.com/endpoints.instances: '[{ "prometheus_url": "https://%%host%%:%%port%%/metrics", "bearer_token_auth": "true" }]'
```

Comment:
if possible we should recommend to use the autodiscovery v2 format for the annotation. thanks to it, it will be only one annotation to add instead of 3. 
see doc: https://docs.datadoghq.com/getting_started/containers/autodiscovery/?tab=adannotationsv2agent736

Reply:
done in 5a607284bb4b0fd49df5d166b8f746655475c019